### PR TITLE
[1LP][RFR] Remove version.pick (part 1)

### PR DESCRIPTION
**Pull request by ohiliazo**
__Updating tests__ to remove support of versions lower than 5.8.

### CODE REVIEW

File: cfme/infrastructure/provider/__init__.py [hunk 1]
```diff
@@ -86,9 +86,7 @@ def __init__(
 
     @cached_property
     def vm_name(self):
-        return version.pick({
-            version.LOWEST: "VMs and Instances",
-            '5.7.1': "Virtual Machines"})
+        return "Virtual Machines"
```
Comment (mfalesni) [inline cfme/infrastructure/provider/__init__.py]: If there are no other instances of `def vm_name`, can you turn this into a class attribute?

File: cfme/infrastructure/config_management.py [hunk 2]
```diff
@@ -360,9 +360,7 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
 
     @property
     def _refresh_flash_msg(self):
-        return version.pick({'5.7': 'Refresh Provider initiated for 1 provider ({})'.
-                                    format(self.type),
-                             '5.8': 'Refresh Provider initiated for 1 provider'})
+        return 'Refresh Provider initiated for 1 provider'
```
Comment (mfalesni) [inline cfme/infrastructure/config_management.py]: This can probably just be a class attribute.

Comment (ohiliazo) [reply cfme/infrastructure/config_management.py]: OK

File: cfme/infrastructure/provider/__init__.py (hunk 1 above)
Comment (ohiliazo) [reply cfme/infrastructure/provider/__init__.py]: OK, I'll look and turn it if everything is ok

Comment (ohiliazo) [reply cfme/infrastructure/provider/__init__.py]: I haven't wound any other instance of `def vm_name` in infrastructure/provider.

File: cfme/infrastructure/config_management.py (hunk 2 above)
Comment (ohiliazo) [reply cfme/infrastructure/config_management.py]: Done.

File: cfme/infrastructure/provider/rhevm.py [hunk 3]
```diff
@@ -71,10 +68,7 @@ def __init__(self, name=None, endpoints=None, zone=None, key=None, hostname=None
     def view_value_mapping(self):
         return {
             'name': self.name,
-            'prov_type': version.pick({version.LOWEST: 'Red Hat Enterprise Virtualization Manager',
-                                       '5.7.1': 'Red Hat Virtualization Manager',
-                                       '5.7.3': 'Red Hat Virtualization',
-                                       '5.8': 'Red Hat Virtualization Manager',
+            'prov_type': version.pick({version.LOWEST: 'Red Hat Virtualization Manager',
```
Comment (vatsalparekh) [inline cfme/infrastructure/provider/rhevm.py]: @ohiliazo in all 5.7z, 5.8z & 5.9z, I see `Red Hat Virtualization` in the provider list, my tests related to RHV failed for 5.7 so got here, maybe you can update version from `5.8.0.10` to latest 5.8z
@psav I think this got merged without running PRT, so this got in without failing.

**Pull request merged**